### PR TITLE
Feature/dashboard links

### PR DESCRIPTION
**Pull request by Bluesmile82**
This PR creates the links from some of the nodes in the dashboard widgets (ranking and horizontal bar chart widgets) to their correspondent profile page. 

If the meta doesn't have a profile attribute that will mean that the node doesn't have a link (eg. states, biomes, ...) and it won't be hoverable.

We still need another PR to create an SVG tooltip that will show a "Go to profile" text and the full name of the node in case it has an ellipsis

image
image

### CODE REVIEW

**Comment by sorodrigo** [comment]
Screen Shot 2019-05-13 at 17 01 12

I wonder if it's possible to add the link to multiyear charts too. Perhaps in the legend?

File: frontend/scripts/react-components/shared/tooltip/tooltip.component.jsx [hunk 1]
```diff
@@ -0,0 +1,56 @@
+import React, { useState } from 'react';
+import PropTypes from 'prop-types';
+import './tooltip.scss';
+import { Manager, Reference, Popper } from 'react-popper';
+
+function Tooltip({ reference, referenceClassName, children, type }) {
```
Comment (sorodrigo) [inline frontend/scripts/react-components/shared/tooltip/tooltip.component.jsx]: is this file necessary?

**Comment by sorodrigo** [comment]
Looks good see my comments and there's a test that needs fixing.

File: frontend/scripts/react-components/shared/tooltip/tooltip.component.jsx (hunk 1 above)
Comment (Bluesmile82) [reply frontend/scripts/react-components/shared/tooltip/tooltip.component.jsx]: I'm afraid yLabelsProfileInfo is not available from the backend in multiyear charts 😕

**Comment by Bluesmile82** [comment]
> I wonder if it's possible to add the link to multiyear charts too. Perhaps in the legend?

I'm afraid yLabelsProfileInfo is not available from the backend in multiyear charts 😕

**Comment by sorodrigo** [comment]
Ok ticket to pivotal, fix test, and lets merge.
